README.md

```diff
@@ -16,6 +16,12 @@ Features:
 or
 `npm install rollup-plugin-handlebars-plus --save`
 
+Handlebars is a peerDependency, so you should install your prefered version like:
+
+`npm install handlebars`
```

Comment:
Curious, is this guaranteed to work? (Shouldn't we suggest the peer dependency range here?)

Reply:
After my modification, if you install `rollup-plugin-handlebars-plus`, you will get the following message:
```
npm WARN rollup-plugin-handlebars-plus@0.3.0 requires a peer of handlebars@>=3.0.0 < 5 but none is installed. You must install peer dependencies yourself.
```
I do not thing we should over-explain this. This is not the first plugin that uses peer dependency, so I think the developers are familiar with this.

Reply:
I'm not sure about this looking at https://github.com/pcardune/handlebars-loader/issues/179. I think it would be better to overcommunicate how to get the library working.

Reply:
We can also see from https://github.com/mixmaxhq/rollup-plugin-handlebars-plus/pull/14#discussion_r285370137 that this library is not guaranteed to work with all versions of Handlebars, maybe even not all versions going forward.